flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergFilesCommitter.java

```diff
@@ -204,7 +207,7 @@ public void testCommitTxn() throws Exception {
       for (int i = 1; i <= 3; i++) {
         RowData rowData = SimpleDataUtil.createRowData(i, "hello" + i);
         DataFile dataFile = writeDataFile("data-" + i, ImmutableList.of(rowData));
-        harness.processElement(of(dataFile), ++timestamp);
+        harness.processElement(of(i, dataFile), ++timestamp);
```

Comment:
Iceberg coding style discourages use `++` for side effect like this.

I know this is not introduced by this PR. just of carrying over of the existing code. so we don't need to fix it in this PR.

Reply:
Thanks for your guidance.